.github/workflows/ci.yml

```
@@ -9,8 +9,8 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-        python-version: [3.9, 3.10]
+        os: [ubuntu-latest, mac-os-12]
```

```suggestion
        os: [ubuntu-latest, macos-latest]
```
Let's try `macos-latest`, I think `macos-12` should work also.   Not sure it should be sensitive to upper cases, though.  So maybe something else is off.